### PR TITLE
Styled landing, added a navbar, created a login page

### PR DESCRIPTION
**Pull request by aemann2**
Forgive me if it's messy -- this is the first thing I've ever done in React and I don't quite know what I'm doing yet. I worked off what Michael did in his branch, but only focused on the login page. I basically copied the login page found below and read up on everything I didn't understand to learn how things work:

https://serverless-stack.com/chapters/create-a-login-page.html

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+module.exports = {
```
Comment (edumqs) [inline .eslintrc.js:1]: There's already a .eslintrc file at the root of the project, can u check if it works for you?

File: clientSrc/components/Login.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+// eslint-disable-next-line no-unused-vars
```
Comment (edumqs) [inline clientSrc/components/Login.js:1]: I would avoid ignoring things like this. If it's not used it's better to remove.

File: clientSrc/styles/Login.less [hunk 3]
```diff
@@ -0,0 +1,28 @@
+
+.Login form {
+  margin: 0 auto;
+  font-size: 1.5em;
+  border-radius: 6px;
+  max-width: 320px;
+  background: rgba(255, 250, 240, .9);
+  box-sizing: content-box;
+  padding: 2em;
```
Comment (edumqs) [inline clientSrc/styles/Login.less:9]: Stick to a single unit for consistency, unless there's a good reason for it

File: package.json [hunk 4]
```diff
@@ -46,7 +47,10 @@
     "path": "0.12.7",
     "prop-types": "15.6.2",
     "react": "16.10.2",
+    "react-bootstrap": "^1.4.0",
+    "react-bootstrap-validation": "^0.1.11",
```
Comment (edumqs) [inline package.json:51]: I think this dependency is not used

File: clientSrc/styles/Login.less (hunk 3 above)
Comment (aemann2) [reply clientSrc/styles/Login.less:9]: Got it -- I think we're going to pull out the Bootstrap eventually, but I'll try to stick with the same units when possible

File: .eslintrc.js (hunk 1 above)
Comment (aemann2) [reply .eslintrc.js:1]: Yes, it worked when I reimported it. Thanks!

**Comment by bryanurizar** [comment]
As suggested, closing as we'll be using Michael's PR!

**Pull request closed without merging**